modules/ocf_desktop/templates/sudoers.d/broker

```diff
@@ -0,0 +1,2 @@
+ALL ALL=(ocfbroker) NOPASSWD: /opt/share/puppet/print-notify-handler
+ALL ALL=(ocfbroker) NOPASSWD: /bin/kill
```

Comment:
I'd definitely feel more comfortable if the `ocfbroker` user didn't have the ability to kill any process. Could the handler script check some file or something to terminate instead?

Reply:
...This gives any user the ability to kill processes running as ocfbroker.
Still, it'd be better to narrow this ability...

Reply:
Other way around. Users can kill as ocfbroker. That said, we could have the handler process clean up and die on BrokenPipeError which would remove the need for all this. We would just need the handler to send 'alive' messages to actually receive that error without getting a print notification. This also wouldn't work if I add a sleep to the while true.

Reply:
Also, as a general note, files that don't use Puppet/ERB template features should go in `files` rather than `templates`. I figure you probably know this already, but would just like to remind you...